lib/app/service/vendor.go

```diff
@@ -329,10 +329,25 @@ func printResourceStatus(resourceFile resources.ResourceFile, manifestPath strin
 	if len(extractedImages.Images) == 0 {
 		progressReporter.PrintSubStep("Found no images to vendor in %v %v", resourceType, relPath)
 	}
+	if len(extractedImages.UnrecognizedObjects) > 0 {
+		progressReporter.PrintSubWarn("Some resources in %v %v weren't recognized, check --debug output",
```

Comment:
So I'm not a huge fan of suppressing the resource warnings behind the debug flag. I interpret the debug UX pattern as I want to troubleshoot tele's behaviour, and get a bunch of debug info about tele itself to troubleshoot, not that this is a pattern that should be used for digging through and debugging an application.

My 2 cents, is I think it's better to print the output like we were doing before, but now that the unrecognized object stuff is fixed, just make sure it's clean, including the filename, line number if possible, etc. If you really want to hide it, then a UX pattern more like `tele build --verbose` for better application debug information may be warranted.

Reply:
Yeah, I wanna hide it by default b/c otherwise people who have a lot of custom resources will be constantly seeing this and even with cleaned up output, these can amount to a lot of noise.

I like the idea of having `--verbose` flag that will print more detailed info including these warning, can do that too.